frontend/src/metabase/embedding-sdk/theme/embedding-color-palette.ts

```diff
@@ -51,7 +51,7 @@ export const SDK_TO_MAIN_APP_COLORS_MAPPING: Record<
   "text-primary": ["text-dark", "text-primary"],
   "text-secondary": ["text-medium", "text-secondary"],
   "text-tertiary": ["text-light", "text-tertiary"],
-  background: ["bg-white", "background"],
+  background: ["bg-white", "bg-primary", "background"],
```

Comment:
We have to map `background` to `bg-primary` otherwise the hover card's background colors will be broken on the SDK:

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/ba76ac32-743b-46b2-9a37-ac118f39ce5d" />